src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.Helpers/Commands.RecoveryServices.Backup.Helpers.csproj

```diff
@@ -40,9 +40,8 @@
     <Reference Include="Microsoft.Azure.Common">
```

Comment:
If you've updated the latest DLL, can you keep this change local? We have another payload that needs to go upstream in the upcoming release.